test/jdk/sun/security/krb5/auto/W83.java

```diff
@@ -23,16 +23,16 @@
 
 /*
  * @test
- * @bug 6932525 6951366 6959292 8194486
+ * @bug 6932525 6951366 6959292 8194486 8139348
```

Comment:
I don't think you should add the bugid here since it isn't really about testing this change.

Reply:
OK, so as long as a test is not meant to test the change for a bug, then its bugid should not appear there.